.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2023/06 to 2023/07
+1. AIRAC (2307) Updated Compton Abbas (EGHA) ARP - thanks to @stu612 (Stuart Duncan)
```

Comment:
```suggestion
1. AIRAC (2307) - Updated Compton Abbas (EGHA) ARP - thanks to @stu612 (Stuart Duncan)
```
Very minor styling thing that just allows us to use some software to auto-parse this at release time :)